modules/osdk-control-compat.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-working-bundle-images.adoc
+
+[id="osdk-control-compat_{context}"]
+= Controlling Operator compatibility with {product-title} versions
+
+[IMPORTANT]
+====
+Kubernetes periodically deprecates certain APIs that are later removed in subsequent releases. As a result, your Operator project will be unable to use any removed APIs starting with the version of {product-title} that uses whichever Kubernetes version removed the API.
+
+As an Operator author, it is strongly recommended that you review the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/[Deprecated API Migration Guide] in Kubernetes documentation and ensure that your projects stay updated to avoid using deprecated and removed APIs, ideally before the release of a future version of {product-title} that would make the Operator incompatible.
+====
```

Comment:
Can "stay updated" be "are updated" or maybe "stay up to date."

If you have time and patience, maybe start a new sentence at "ideally."  Maybe, "Ideally, you can update your Operator before..."

Reply:
✅
✅